.vscode/settings.sample.json

```diff
@@ -0,0 +1,7 @@
+{
+  "jest.autoRun": {
+    "watch": false,
+    // "onSave": "test-file"
+  },
+  "jest.jestCommandLine": "yarn test"
```

Comment:
Tell vscode to use `yarn test` to run the test. Without this, vscode will run the test using the default setting which is not compatible with this repo because the repo has custom jest config (the jest config file is in `test/jest.config.js` instead of the default `<root_dir>/jest.config.js`). 

Here's a preview when we disable/enable the `jest.jestcommandLine` (when disabled, vscode couldn't detect the test):

https://user-images.githubusercontent.com/7823011/188500720-afcd08f3-f221-4899-ab88-28099e18756b.mov

Note:
- make sure you've installed [vscode jest extension](https://marketplace.visualstudio.com/items?itemName=Orta.vscode-jest).